README.md

```diff
@@ -1,2 +1,36 @@
-# maintt-frontend
-Frontend for Maintt - https://github.com/AndelaOSP/maintt
+[![gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/AndelaOSP/maintt-frontend?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
You prefer us moving to Gitter or we could just maintain the current channel on our Slack Org?